kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -237,7 +239,7 @@ protected final Optional<List<FileStatus>> listDeltaAndCheckpointFiles(
      * Load the Snapshot for this Delta table at initialization. This method uses the
      * `lastCheckpoint` file as a hint on where to start listing the transaction log directory.
      */
-    private SnapshotImpl getSnapshotAtInit(TableClient tableClient, Path logPath, Path dataPath)
```

Comment:
I thought this internal method made sense to remove from arg list. I'm good with whatever on the rest (would be fine to remove from any and all instance methods).

Reply:
We should remove all data path and log path params from _instance_ methods part of this class. They are all for the same table, same data path, same log path

Reply:
Updated but seems weird to not have `path` as an argument to methods like `listFrom`. Maybe these should be static methods